bin/init-release

```diff
@@ -5,62 +5,56 @@ releaseVersion=$1
 masterTag=$2
 hasTag=$(git tag -l "$masterTag" | wc -l | tr -d '[:space:]')
 
-
-# Helper function to display text in yellow.
-function log () {
-  echo -e "\033[33m"
-  echo "$1"
-  echo -e "\033[0m"
-}
-
+currentDir=$(node -e "console.log(require('path').dirname(require('fs').realpathSync('$BASH_SOURCE')))")
```

Comment:
What is the reason of this? Is it because `li-release` is in the `bin` field of the *package.json*?